### DISCUSSION TITLE
怎么设置才能让道中船捡弹药？

### DISCUSSION THREAD

**Discussion by NafixWang** [LmeSzinc/AzurLaneAutoScript#1144] [Q&A] [answered]
我发现单队出击的时候道中船似乎并不会捡弹药 即使在关闭了自律寻敌以后也并不会 
0弹药的debuff高达-50%啊！！！

**Comment by LmeSzinc** [comment, marked as answer]
12-14图关掉自律寻敌，alas就会帮拣弹药、拣照明弹、开灯塔等
低级图不拣，反正都是乱杀，拣了还浪费时间
